pipelines/src/main/scala/com/fulcrumgenomics/str/pipelines/PostDuplexConsensusPipeline.scala

```diff
@@ -0,0 +1,113 @@
+  @arg(flag="r", doc="Path to the reference FASTA.")              val ref: PathToFasta,
+  @arg(flag="o", doc="Path prefix for output files.")             val output: PathPrefix,
+  @arg(flag="N", doc="Mask (make 'N') consensus bases with quality less than this threshold.")       val minConsensusBaseQuality: Option[Int] = Some(50),
+  @arg(flag="M", minElements=1, maxElements=3, doc="The minimum number of reads supporting a consensus base/read.")
+  val minReads: Seq[Int] = Seq(6,3,3),
+  @arg(flag="E", minElements=1, maxElements=3, doc="The maximum raw-read error rate across the entire consensus read.")
+  val maxReadErrorRate: Seq[Double] = Seq(0.05),
+  @arg(flag="e", minElements=1, maxElements=3, doc="The maximum error rate for a single consensus base.")
+  val maxBaseErrorRate: Seq[Double] = Seq(0.1),
+  @arg(flag="n", doc="Maximum fraction of no-calls in the read after filtering.")
+  val maxNoCallFraction: Double = 0.2,
+  @arg(flag="f", doc="The minimum allele frequency for variant calling") val minimumAf: Double = 0.0001
+)
+  extends Pipeline(Some(output.getParent)) {
+
+  Io.assertReadable(input)
+  Io.assertCanWriteFile(output, parentMustExist=false)
+
+  def build(): Unit = {
+    // Files that we're going to create
+    val outDir      = output.getParent
+    val inDir       = input.getParent
+    val dsMapped    = inDir.resolve(input.getFileName + ".consensus.mapped.bam")
+    val dsFiltered  = outDir.resolve(output.getFileName + ".consensus.filtered.bam")
+
+    // Copy the intervals to the directory and make a BED file
+    val regionsIl     = inDir.resolve("calling_regions.interval_list")
+    val regionsBed    = inDir.resolve("calling_regions.bed")
+
+    val filterDs = FilterConsensusReads(in=dsMapped, out=dsFiltered, ref=ref, minReads=Seq(6,3,3), maxReadErrorRate=Seq(0.05),
+      maxBaseErrorRate=Seq(0.1), minQuality=minConsensusBaseQuality.getOrElse(0), maxNoCallFraction=0.05).requires(cores=1, memory="8G")
+
+    // Make variant calls and assess the results
+    Seq((dsFiltered, filterDs)).foreach { case(bam, previous) =>
+      val clippedBam = PathUtil.replaceExtension(bam, ".clipped.bam")
+      val clip = new ClipOverlappingReads(in=bam, out=clippedBam, ref=ref).requires(cores=1, memory="8G")
+      val prior = previous ==> clip
+
+      // Things we do to _every_ bam
+      prior ==> new CollectAlignmentSummaryMetrics(in=clippedBam, ref=ref)
+      prior ==> new CollectHsMetrics(in=clippedBam, ref=ref, targets=regionsIl)
+      prior ==> new CollectSequencingArtifactMetrics(in=clippedBam, ref=ref, intervals=Some(regionsIl), minBq=Some(30), contextSize=Some(0), variants=truthVcf)
+
+      // Do some variant calling
+      val vcf = PathUtil.replaceExtension(bam, ".vcf")
+      val vardict = prior ==> new VarDictJavaEndToEnd(tumorBam=clippedBam, tumorName=Some("tumor"), bed=regionsBed, ref=ref, out=vcf, minimumAf=0.0025, minimumQuality=Some(30))
```

Comment:
I want it there for now to see the results of a non-STR caller.  I think I'll make it output pileup mode while I am at it.